packages/expo-notifications/src/index.ts

```diff
@@ -1,6 +1,10 @@
 import { isRunningInExpoGo } from 'expo';
 
-if (isRunningInExpoGo()) {
+function areWeTestingWithJest() {
+  return process.env.JEST_WORKER_ID !== undefined || process.env.NODE_ENV === 'TEST';
```

Comment:
just curious: any reason for the `JEST_WORKER_ID` check?

Reply:
The `isRunningInExpoGo()` call returns true when we are doing Jest testing.